BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+"""Root configuration for Bazel build tool.
+"""
+
+# Allow the tsconfig.json file to be referenced by subpackages
+exports_files(["tsconfig.json"])
+
+# Provides the npm-installed dependencies to Bazel actions.
+filegroup(
+    name = "node_modules",
```

Comment:
So we're declaring this here with `name="node_modules"` because some other Bazel rules, like the `ng_module` or some other rule requires it with this name in order to have access to the `node_modules`?

Reply:
right, the default value for `ts_library` and `nodejs_binary` is to use the `//:node_modules` target in the workspace where the build is running.

I'd like to make this configurable soon, so you could have it in a subdirectory for example.